CMakeLists.txt

```diff
@@ -255,9 +255,11 @@ IF (WIN32 AND WINHTTP)
 
 		INCLUDE_DIRECTORIES(deps/winhttp)
 		LINK_DIRECTORIES(${LIBWINHTTP_PATH})
+		LIST(APPEND LIBGIT2_PC_LIBS "-L${LIBWINHTTP_PATH}")
```

Comment:
I dropped this line again. the winhttp stub is only generated because mingw is missing an import library for winhttp, and the generated file/path will only be available during the compile phase of libgit2